engine/access/rpc/connection/connection_test.go

```diff
@@ -749,9 +728,7 @@ func TestEvictingCacheClients(t *testing.T) {
 
 	// Call a gRPC method on the client
 	_, err = client.Ping(ctx, pingReq)
-	// Check that Ping was called
-	cn.handler.AssertCalled(t, "Ping", testifymock.Anything, pingReq)
```

Comment:
these asserts are called automatically now for the collection and execution mock clients